modules/data/webadmin/tmpl/add_edit_network.tmpl

```diff
@@ -31,8 +31,9 @@
 			<div class="sectionbody">
 				<div class="subsection">
 					<div class="inputlabel"><label for="name"><? FORMAT "Network Name:" ?></label></div>
-					<input id="name" type="text" name="name" value="<? VAR Name ?>" class="half" maxlength="128"
+					<input id="name" type="text" name="name<? IF !NameEdit ?>_disabled<? ENDIF ?>" value="<? VAR Name ?>" class="half" maxlength="128"
 						   title="<? FORMAT "The name of the IRC network." ?>" <? IF !NameEdit ?>disabled<? ENDIF ?> />
+					<? IF !NameEdit ?><input type="hidden" name="name" value="<? VAR Name ?>"><? ENDIF ?>
```

Comment:
What's the point of adding the hidden field AND renaming the old field?

I think the change you did in .cpp should be enough

Reply:
The browser will not send disabled fields on submit, so it will be missing in the POST request and with error message `Network name is a required argument`, that's why a hidden field was added to transfer this field. The disabled input field was renamed because of HTML compliance (input name's in the same form ought to be unique).